.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Please feel free to remove whatever sections/lines in this aren't relevant. -->
+
+## Title Line Template: [Website] - [UI components]: [Brief description.]
+<!-- Use the title line as the title of your pull request, then delete these lines. 
+
+Website: Issues that impact standards.usa.gov look, feel or functionality.
+UI components: Issues that impact the look, feel or functionality of the standards themselves.
+
+-->
+
+## Description
```

Comment:
Will we need Issue templates for the different types of issues we're looking to track/have? The way this one is currently written seems to be more for a bug. What would an issue for a new component or enhancement look like?